src/Xive/Hive/MutexHive.cs

```diff
@@ -108,7 +108,7 @@ private void Block()
                 
                 if (this.mtx.Count == 0)
                 {
-                    this.mtx.Add(new Mutex(false, $"Local/{this.hive.Scope()}"));
```

Comment:
I think the correct fix, according to the msdn docs, would be to change the / to a \ - please do this and check if the linux build works.

See: 

> "On a server that is running Terminal Services, a named system mutex can have two levels of visibility. If its name begins with the prefix "Global\", the mutex is visible in all terminal server sessions. If its name begins with the prefix "Local\", the mutex is visible only in the terminal server session where it was created. In that case, a separate mutex with the same name can exist in each of the other terminal server sessions on the server. If you do not specify a prefix when you create a named mutex, it takes the prefix "Local\". Within a terminal server session, two mutexes whose names differ only by their prefixes are separate mutexes, and both are visible to all processes in the terminal server session. That is, the prefix names "Global\" and "Local\" describe the scope of the mutex name relative to terminal server sessions, not relative to processes."

https://docs.microsoft.com/en-gb/dotnet/api/system.threading.mutex?view=netframework-4.8

Reply:
I tried that. It doesn't work.